src/easylink/steps/rl-dummy/eliminating_records/dummy_eliminating_records.py

```diff
@@ -0,0 +1,104 @@
+import glob
+import logging
+import os
+
+import pandas as pd
+import yaml
+
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s %(message)s",
+    handlers=[logging.StreamHandler()],
+)
+
+
+def load_file(file_path, file_format=None):
+    if file_format is None:
+        file_format = file_path.split(".")[-1]
+    if file_format == "parquet":
+        return pd.read_parquet(file_path)
+    if file_format == "csv":
+        return pd.read_csv(file_path)
+    raise ValueError()
+
+
+INPUT_ENV_VARS = os.getenv(
+    "INPUT_DATASETS_FILE_PATHS",
+    "KNOWN_CLUSTERS_FILE_PATHS",
+).split(",")
+
+diagnostics = {}
+
+for env_var in INPUT_ENV_VARS:
+    if env_var not in os.environ:
+        logging.error(f"Missing required environment variable {env_var}")
+        raise ValueError()
+
+# LOAD INPUTS
+
+# Load datasets
+datasets_var = INPUT_ENV_VARS[0]
```

Comment:
This should be just `datasets_var = os.environ["INPUT_DATASETS_FILE_PATHS"]`. Then we can delete the `INPUT_ENV_VARS` part.